src/action.ts

```diff
@@ -24,6 +24,8 @@ import * as yaml from 'js-yaml'
 
 export async function run () {
   try {
+    process.env.GNUPGHOME = process.env.RUNNER_WORKSPACE
```

Comment:
This doesn't help much, because you are replacing the static `/root` directory with the also static workspace dir of the runner. This will not change between executions of workflow steps.

Reply:
Runner workspace changes for every repository. For eg., for a repository called `repository1` it will be
`/tmp-github-runner/runner/repository1`.
So, this should help in solving the static location for GPG for different repositories. 
Also, I think we should not tie this up with runner's setup because this could also create a problem if this action is used in any other organization where the runner setup is different.

Reply:
![runner-env](https://user-images.githubusercontent.com/13904552/116884441-07d47c00-ac27-11eb-8dc0-664d5a2bcd63.png)

Yes, you are right.